apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/FissClaimTransformerV2.java

```diff
@@ -88,21 +131,56 @@ private static Claim transformClaim(PreAdjFissClaim claimGroup) {
                         C4BBClaimIdentifierType.UC.getSystem(),
                         C4BBClaimIdentifierType.UC.toCode(),
                         C4BBClaimIdentifierType.UC.getDisplay())))
-            .setSystem("https://dcgeo.cms.gov/resources/variables/dcn")
+            .setSystem(BBCodingSystems.FI_DOC_CLM_CONTROL_NUM)
             .setValue(claimGroup.getDcn()));
   }
 
+  private static List<Extension> getExtension(PreAdjFissClaim claimGroup) {
+    return claimGroup.getServTypeCd() == null
```

Comment:
@KyRobbins sorry about this, but I just found an error in the example JSON files (its correct in the confluence table though).  The CLM_SERVICE_CLSFCTN_TYPE_CODE should appear in the `Claim.type` list.

Reply:
I spoke with Bill about this more this morning.  While the ServTypeCd field does provide some type information, its not an exact match for what eob-type or the nch claim type field provides, so like the EOB we decided this should be moved back to an extension.

I reverted the previous commit to resolve this.